flight-icons/scripts/sync.ts

```diff
@@ -37,6 +37,16 @@ import { config } from './config';
 async function sync() {
 
     // remove existing output folder
+    /* 
+        Question: do we need to run this every time, or could we just overwrite icons that have changed? 
+        No strong opinion, just curious of the thought here. 
```

Comment:
Icons can also be removed, or renamed, not just added (rare, but it happens). That's why it's easier to just delete the output folder and don't have special logic to handle that.